src/main/java/org/generationcp/middleware/api/germplasmlist/GermplasmListService.java

```diff
@@ -96,4 +96,6 @@ void addGermplasmEntriesToList(Integer germplasmListId, SearchCompositeDto<Germp
 
 	void addGermplasmListEntriesToAnotherList(Integer destinationListId, Integer sourceListId, String programUUID,
 		SearchCompositeDto<GermplasmListDataSearchRequest, Integer> searchComposite);
+
+	void removeGermplasmEntriesFromList(Integer germplasmListId, List<Integer> listDataIds);
```

Comment:
For flexibility and uniformity with other methods (ie. to add entries to list, add entries to another list) - better to use `SearchCompositeDto<GermplasmSearchRequest, Integer> searchComposite` as 2nd parameter to specify which entries to delete